Models.swift

```diff
@@ -177,4 +177,27 @@ struct User: Codable {
         temp = temp + (lastName ?? "")
         return temp
     }
+    
+    var idString: String? {
+        id == nil ? nil : String(id!)
+    }
+}
+
+// MARK: - PreferenceDataResponse
+struct PreferenceDataResponse: Codable {
+    let newBookMark: [String : Bool]
+    let newPost: [String: Bool]
+    
+    var newBookMarkNotificationEnabled: Bool {
+        newBookMark["enabled"] ?? false
+    }
+    
+    var newPostNotificationEnabled: Bool {
+        newPost["enabled"] ?? false
+    }
```

Comment:
Similar to Android, shouldn't we add a `Preference` class that is exposed by SDK which contains the keys `enabled`, `email`, `sms`, etc?

Reply:
Yeah, we can do that. Will update